templates/tfengine/components/cicd/triggers/validate/main.tf

```diff
@@ -13,7 +13,7 @@ See the License for the specific language governing permissions and
 limitations under the License. */ -}}
 locals {
   terraform_root = var.terraform_root == "/" ? "." : var.terraform_root
-  terraform_root_prefix = var.terraform_root == "/" ? "" : "${var.terraform_root}/"
+  terraform_root_prefix = local.terraform_root == "." ? "" : "${var.terraform_root}/"
```

Comment:
can we also use local.terraform in the "${local.terraform_root}/" as well? I think logically it won't make a difference, but more consistent if we use one terraform_root in terraform_root_prefix value definition.

Reply:
Sure, looks better to me too.

Done